src/lib/frontmatter.ml

```diff
@@ -69,13 +69,9 @@ let input_map t = t.inputs
 
 let default_container_path t =
   let default = "/root" in
-  let path = match (List.assoc_opt "path" t.variables) with
-  | None -> default
-  | Some pl -> (
-    match (List.nth_opt pl 0) with
+  let path =
+    match List.assoc_opt "path" t.variables with
     | None -> default
-    | Some p -> p
-  ) in
-  match Fpath.of_string path with
-  | Error (`Msg m) -> failwith m
-  | Ok p -> p
+    | Some pl -> ( match List.nth_opt pl 0 with None -> default | Some p -> p)
```

Comment:
I think it would be cleaner (and probably more efficient) to use pattern-matching here directly?

```suggestion
    | Some [] -> default
    | Some (p :: _) ->  p
```